Mage/src/main/java/mage/util/CardUtil.java

```diff
@@ -1186,8 +1187,24 @@ public static void takeControlUnderPlayerEnd(Game game, Player controller, Playe
         }
     }
 
-    public static void makeCardPlayable(Game game, Ability source, Card card, Duration duration, boolean anyColor) {
-        makeCardPlayable(game, source, card, duration, anyColor, null, null);
+    /**
+     * Groups together the most usual ways a card's payment is adjusted
+     * by card effects that allow play or cast.
+     */
+    public enum SimpleCastManaAdjustment {
+        AS_THOUGH_ANY_MANA_TYPE,
+        AS_THOUGH_ANY_MANA_COLOR,
+        WITHOUT_PAYING_MANA_COST,
```

Comment:
I think it would be better to have a `NONE` option in the enum rather than using `null` as an option.

(Also maybe leave out the word "simple" to shorten it.)